<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseTestData</class>
 <widget class="QDialog" name="chooseTestData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="liTests">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtTestInfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bChooseTest">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Testi Se√ß</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="testPreviewGraph" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>391</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bChooseTest</sender>
   <signal>clicked()</signal>
   <receiver>chooseTestData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
